app/server/app-configurations.ts

```diff
@@ -35,8 +35,7 @@ const contentSecurityPolicy: ContentSecurityPolicyOptions = {
       'tagmanager.google.com',
       'vcc-eu4.8x8.com',
       'https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.min.js',
-      'https://code.jquery.com/ui/1.12.1/jquery-ui.js',
-      'https://code.jquery.com/jquery-3.6.0.js',
+      'https://code.jquery.com/jquery-3.7.1.js',
```

Comment:
is this actually needed in MYA?

Reply:
I didn't mean the update, I meant the whitelisting of this jQuery url.
It was done in SYA because they had an inline script referencing jQuery, but I didn't see one in MYA